test/org/zaproxy/zap/extension/pscanrulesAlpha/TimestampDisclosureScannerUnitTest.java

```diff
@@ -0,0 +1,44 @@
+package org.zaproxy.zap.extension.pscanrulesAlpha;
+
+import org.apache.commons.httpclient.URI;
+import org.apache.commons.httpclient.URIException;
+import org.junit.Before;
+import org.junit.Test;
+import org.parosproxy.paros.network.HttpMessage;
+import org.parosproxy.paros.network.HttpRequestHeader;
+import org.zaproxy.zap.extension.pscan.PluginPassiveScanner;
+
+import static org.junit.Assert.assertEquals;
+
+public class TimestampDisclosureScannerUnitTest extends PassiveScannerTest  {
+    private HttpMessage msg;
+
+    // Hashes in lower case for "guest" without quotes
```

Comment:
These lines can be removed.

Reply:
Sorry, should pay more attention when I copied the code...

Reply:
No worries, been there done that myself :zap: